build/package/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# syntax=docker/dockerfile:1
+
+FROM golang:1.22-alpine
+
+# Set the working directory inside the container
+WORKDIR /app
+
+# Install necessary packages including OpenSSL and CA certificates
+RUN apk --no-cache add openssl ca-certificates
+
+# Copy the go.mod and go.sum files to the work directory
+COPY go.mod go.sum ./
+
+# Generate certificates
+COPY scripts ./scripts
+RUN chmod +x ./scripts/generate_certs.sh
+RUN sh ./scripts/generate_certs.sh
```

Comment:
As far as I see `generate_certs.sh` generates both server and client certificates.
While it's clear why we need server certs inside container, do we need to keep client ones?